### ISSUE TITLE
Strange condition: checking if acos(U(3,3)) == 1 

### ISSUE THREAD

**Issue by jkshenton** [LeoSvenningsson/TensorViewforMatlab#2]
Hi! 

Thanks for making this very useful code available! 

I am currently implementing a similar thing in python and using your code as the reference. However, I came across these lines:

https://github.com/LeoSvenningsson/TensorViewforMatlab/blob/1d0bc6d0d3dfb7f599a780519429a0ae0ee86de8/Source/functions/MFtoEuler.m#L188

https://github.com/LeoSvenningsson/TensorViewforMatlab/blob/1d0bc6d0d3dfb7f599a780519429a0ae0ee86de8/Source/functions/MFtoEuler.m#L323

https://github.com/LeoSvenningsson/TensorViewforMatlab/blob/1d0bc6d0d3dfb7f599a780519429a0ae0ee86de8/Source/functions/MFtoEuler.m#L459

that seems strange to me - is it correct? Shouldn't it be checking if `U(3,3) == 1`?

Thanks!

**Comment by LeoSvenningsson** [comment]
You are right. it should be U(3,3) == 1. I will change it, though i dont have an option to recompile the "app" at the moment. Since you are making something similar. Let me know if you have any questions, the tryallanglestest.m for example could be a bit challenging why its done this way. Otherwise, you can suggest other code improvements as well. There are quite a few ways to make the code more neat.

**Comment by jkshenton** [comment]
Thanks for checking and updating the code. I noticed the bug while trying to cross-reference some edge-case tests for our Python code [Soprano](https://github.com/CCP-NC/soprano/blob/nmr_conventions/soprano/nmr/tensor.py). Not knowing anything about MATLAB, I think the option of having some tests of TensorView, particularly for the tricky cases that your paper deals with so comprehensively, would be extremely useful! 

Thanks for the offer of help - I'll definitely come back with more questions :)
